docs/framework/wcf/feature-details/routing-introduction.md

```diff
@@ -359,21 +359,21 @@ rc.FilterTable.Add(new MatchAllMessageFilter(), backupList);
 The following table describes the patterns that are compatible with the use of backup endpoint lists, along with notes describing the details of error handling for specific patterns.
 
 |Pattern|Session|Transaction|Receive Context|Backup List Supported|Notes|
-|-------------|-------------|-----------------|---------------------|---------------------------|-----------|
+|:-:|:-:|:-:|:-:|:-:|-|
 |One-Way||||Yes|Attempts to resend the message on a backup endpoint. If this message is being multicast, only the message on the failed channel is moved to its backup destination.|
-|One-Way||✓||No|An exception is thrown and the transaction is rolled back.|
-|One-Way|||✓|Yes|Attempts to resend the message on a backup endpoint. After the message is successfully received, complete all receive contexts. If the message is not successfully received by any endpoint, do not complete the receive context.<br /><br /> When this message is being multicast, the receive context is only completed if the message is successfully received by at least one endpoint (primary or backup). If none of the endpoints in any of the multicast paths successfully receive the message, do not complete the receive context.|
-|One-Way||✓|✓|Yes|Abort the previous transaction, create a new transaction, and resend all messages. Messages that encountered an error are transmitted to a backup destination.<br /><br /> After a transaction has been created in which all transmissions succeed, complete the receive contexts and commit the transaction.|
-|One-Way|✓|||Yes|Attempts to resend the message on a backup endpoint. In a multicast scenario only the messages in a session that encountered an error or in a session whose session close failed are resent to backup destinations.|
-|One-Way|✓|✓||No|An exception is thrown and the transaction is rolled back.|
-|One-Way|✓||✓|Yes|Attempts to resend the message on a backup endpoint. After all message sends complete without error, the session indicates no more messages and the Routing Service successfully closes all outbound session channel(s), all receive contexts are completed, and the inbound session channel is closed.|
-|One-Way|✓|✓|✓|Yes|Abort the current transaction and create a new one. Resend all previous messages in the session. After a transaction has been created in which all messages have been successfully sent and the session indicates no more messages, all the outbound session channels are closed, receive contexts are all completed with the transaction, the inbound session channel is closed, and the transaction is committed.<br /><br /> When the sessions are being multicast the messages that had no error are resent to the same destination as before, and messages that encountered an error are sent to backup destinations.|
+|One-Way||![check mark icon](../../../media/check-mark.png)||No|An exception is thrown and the transaction is rolled back.|
```

Comment:
In reviewing these specific updates, I'm not sure I see the value in the old check mark nor the icon. Is the point of having these to avoid the YES/NO text? There is already included in another column. Why not just make all of these YES/NO instead of icons? Also, in a table, using icon links does bloat it and make it hard to edit.

Reply:
I believe the point of using an emoji/graphic is to make the table more easily scannable and more visually attractive - which is important for attracting readers.